doc/usage/bibliography.md

```diff
@@ -19,7 +19,7 @@ which contains bibtex entries.
 
 ```{code-cell} ipython3
 from echemdb.cv.cv_database import CVDatabase
-db = CVDatabase()
+db = CVDatabase().create_example()
```

Comment:
currently we cannot use the database entrries, since `website`must be updated to the new version of svgdigitizer, as well as a new version of `echemdb` which contains the current changes.